doc/users/fonts.rst

```diff
@@ -0,0 +1,79 @@
+All about Matplotlib and Fonts!
+===============================
+
+The story of fonts has been quite eventful throughout time. It involves
+contributions of tech giants such as the likes of Adobe, Apple and Microsoft.
+
+Types
+-----
+In practice, there are 3 types Matplotlib supports:
+
+.. list-table:: Type of Fonts
+   :header-rows: 1
+
+   * - Type 1 (PDF)
+     - Type 3 (PDF/PS)
+     - TrueType (PDF)
+   * - One of the oldest types, introduced by Adobe
+     - Similar to Type 1 in terms of introduction
+     - Newer than previous types, most commonly used today, introduced by Apple
+   * - They use simplified PostScript
+       (subset of full PostScript language)
```

Comment:
This may oversimplify the matter a little - the Type 1 wrapper format is a restricted subset of PostScript but the glyph programs ("charstrings") are in a bytecode format that is not really PostScript, although the operators are close to some PostScript operators.

Reply:
Updated this bit